.changelog/28306.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/aws_elasticache_replication_group: Add `ip_discovery` and `network_type` arguments in support of [IPv6 clusters](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/network-type.html)
```

Comment:
No need for the href here to `IPv6 clusters`

Reply:
Removed href